R/SailaniIPOPData.R

```diff
@@ -0,0 +1,95 @@
+#'
+#' @author Sudarshan Shetty and Felix G.M. Ernst
+#'
+#' @references
+#' Sailani, M.R., Metwally, A.A., Zhou, W. et al. (2020) Deep longitudinal multiomics 
+#' profiling reveals two biological seasonal patterns in California.
+#' \emph{Nat Commun.} 11, 4933 \url{https://doi.org/10.1038/s41467-020-18758-1}
+#'
+#' Zhou, W., Sailani, M.R., Contrepois, K. et al. (2019) Longitudinal multi-omics of 
+#' host-microbe dynamics in prediabetes.
+#' \emph{Nature} 569, 663-671 \url{https://doi.org/10.1038/s41586-019-1236-x}
+#'
+#' @name SailaniIPOPData
+#' @export
+#'
+#' @examples
+#' mae <- SailaniIPOPData()
+#' #tse <- SailaniIPOPData()
+SailaniIPOPData <- function() {
+    mae <- .create_mae("sailani-ipop",
+                       types = list(gutMicrobiota = list("TSE" = c("relabundance")),
+                                    nasalMicrobiota = list("TSE" = c("relabundance")),
+                                    hostTranscriptome = list("SE" = c("counts")),
+                                    hostProteome = list("SE" = c("counts")),
+                                    hostMetabolome = list("SE" = c("counts")),
+                                    hostCytokines = list("SE" = c("counts")),
+                                    hostClinical = list("SE" = c("counts"))),
+                       coldata = FALSE,
+                       samplemap = FALSE,
+                       has.rowdata = list(gutMicrobiota = FALSE,
+                                          nasalMicrobiota = FALSE,
+                                          hostTranscriptome = FALSE,
+                                          hostProteome = FALSE,
+                                          hostMetabolome = FALSE,
+                                          hostCytokines = FALSE,
+                                          hostClinical = FALSE),
+                       has.coldata = list(gutMicrobiota = TRUE,
+                                          nasalMicrobiota = TRUE,
+                                          hostTranscriptome = TRUE,
+                                          hostProteome = TRUE,
+                                          hostMetabolome = TRUE,
+                                          hostCytokines = TRUE,
+                                          hostClinical = TRUE))
+    mae
+}
+
+
+# #' @rdname SailaniIPOPData
+# #' @importFrom SummarizedExperiment colData<-
+# #' @export
+
+SailaniIPOPData <- function() {
+    dataset <- "sailani-ipop"
+    hub <- ExperimentHub()
+    tse <- .create_tse(dataset,
+                       hub = hub,
+                       assays = c("relabundance"),
+                       has.rowdata = FALSE,
+                       has.coldata = TRUE) #"$Microbiota"
+    args <- .get_col_row_map_data(dataset,
+                                  hub = hub,
+                                  has.rowdata = FALSE,
+                                  has.coldata = TRUE)
+    colData(tse) <- args$colData
+    tse
+}
```

Comment:
You are over writing the original function `SailaniIPOPData` from lines 44-70 with this.

Reply:
This mostly likely happened when try to sync with microbiome::master with my forked branch. No link between these two datasets. Do you know how i can solve/avoid this issue?

Reply:
Nope. Usually this is not a problem during merging. I don't know, what might have happened here